tests/test_dhcp_relay.py

```diff
@@ -15,6 +15,15 @@ def copy_ptftests_directory(ptfhost):
     """
     ptfhost.copy(src="ptftests", dest="/root")
 
+@pytest.fixture(scope="module", autouse=True)
+def update_ptf_interface_mac_addresses(ptfhost, autouse=True):
+    """ Fixture which copies the change_mac.sh script to the PTF host. This fixture
+        is scoped to the module, as it only needs to be performed once before
+        the first test is run. The fixture also changes mac addresses of ptf containers
+    """
```

Comment:
Can you update the function name, comment and PR description to specify that this function configures the PTF host interfaces have unique MAC addresses?